class/subscription.php

```diff
@@ -1256,4 +1259,55 @@ public function reset_trial( $sub_meta ) {
         }
         return $sub_meta;
     }
+
+    /**
+     * Add daily cron for non recur subscritpion
+     *
+     * @since WPUF_PRO
+     *
+     * @return void
+     */
+    public function handle_non_recur_subs() {
+        if ( ! wp_next_scheduled( 'non_recur_subs_daily' ) ) {
+            wp_schedule_event( time(), 'daily', 'non_recur_subs_daily' );
+        }
+    }
+
+    /**
+     * Cancel non recurring subs if expired
+     *
+     * @since WPUF_PRO
+     *
+     * @return void
+     */
+    public function cancel_non_recurrent_subscription() {
```

Comment:
Seems spelling mistake in the function name. It may be `cancel_non_recurring_subscription` instead of `cancel_non_recurrent_subscription`